apps/condo/domains/property/schema/fields/AddressMetaField.js

```diff
@@ -1,5 +1,11 @@
 const Ajv = require('ajv')
 
+const AddressMetaDataMetroField = {
```

Comment:
Are we sure that we need to use the word `Metro` here? Why not `Underground`? It seems to be more international

Reply:
It is a mapping of fields from third-party service with no extra additions.